docs/release-notes/8.17.asciidoc

```diff
@@ -58,6 +74,22 @@ Affected users who are unable to upgrade should set one or both of the following
 [[known-issue-8.17.0]]
 ==== Known issues
 
+// tag::known-issue[]
+[discrete]
+.{elastic-sec} crashes on {kib} instances with 1 GB of RAM 
```

Comment:
@nastasha-solomon We missed the second note:

```suggestion
.{elastic-sec} crashes on {kib} instances with 1 GB of RAM in {ecloud} deployments
```

Reply:
no need to apologize - I added it in [bd02804](https://github.com/elastic/security-docs/pull/6512/commits/bd0280490131da19c9f2a9115d98b5b4f556a210) : )